developerGuide.t2t

```diff
@@ -228,6 +229,44 @@ Events will be covered in greater detail later.
 
 As with other examples in this guide, remember to delete the created app module when you are finished testing and then restart NVDA or reload plugins, so that original functionality is restored.
 
+++ App modules for hosted apps ++
+Some executables host various apps inside.
+These include javaw.exe for running various java programs and wwahost.exe for web apps in Windows 8 and later.
+
+If an app runs inside a host executable, the name of the app module must be the name as defined by the host executable, which can be found through AppModule.appName property.
+For example, an app module for a java app named "test" hosted inside javaw.exe must be named test.py.
+For apps hosted inside wwahost, not only the app module name must be the name of the loaded app, but the app module must subclass the app module class found in wwahost.

```

Comment:
nit: "not only the app module name must be the name of the loaded app" -> "not only must the app module name be the name of the loaded app" (move the word "must")